packages/block-editor/src/components/global-styles/dimensions-panel.js

```diff
@@ -612,7 +613,11 @@ export default function DimensionsPanel( {
 					hasValue={ hasChildLayoutValue }
 					label={ childLayoutOrientationLabel }
 					onDeselect={ resetChildLayoutValue }
-					isShownByDefault={ defaultControls.childLayout }
+					isShownByDefault={
+						typeof defaultControls.childLayout === 'boolean'
```

Comment:
Why it's not boolean?

Reply:
If I'm understanding properly, the check is less about type "boolean" and more about checking if the value is actually set to avoid considering "undefined" as "false".

I think I'd prefer if we stay consistent between all the default controls. We're considering undefined as false for all the default controls in all the `*Panel` components. So my question is: can we just keep it as is here? and update the "block.json" files for the blocks that override the default defaultControls instead?

Reply:
> If I'm understanding properly, the check is less about type "boolean" and more about checking if the value is actually set to avoid considering "undefined" as "false".

Yes, that's the point of this check.

> We're considering undefined as false for all the default controls in all the *Panel components. So my question is: can we just keep it as is here? and update the "block.json" files for the blocks that override the default defaultControls instead?

That's exactly what this change addresses: we need to have a way of defaulting to true for the child layout controls, but still preserving the ability to override that if needed by setting it to false in block.json. Child layout controls can be applied to any block, including third party blocks, so it's not practical to explicitly set the controls to display for every single block type.

Reply:
I like the idea of exposing `false` as a valid value, and the idea that we can have some controls be opt-out (in terms of the default controls) rather than opt-in.

If consistency for each of the controls is important, what if we used something like `isShownByDefault={ defaultControls.childLayout ?? FALLBACK_DEFAULT_CONTROLS.childLayout }` for each of the controls. Then, `FALLBACK_DEFAULT_CONTROLS` could contain the global defaults we'd like to go with. So if all the others are hidden by default, it might be:

```
const FALLBACK_DEFAULT_CONTROLS = {
	...
	padding: false,
	margin: false,
	blockGap: false,
	minHeight: false,
	childLayout: true,
};
```

Would something like that work? The idea's basically the same as this PR, but just that we could use the same approach as this PR for each of the controls.

Reply:
> isShownByDefault={ defaultControls.childLayout ?? FALLBACK_DEFAULT_CONTROLS.childLayout }

This works for me 👍

Reply:
Thanks for the suggestion! I'll give it a go. Note that doing that will also require us to explicitly set all the controls to `true` on the site editor side, given that currently the site editor isn't passing any `defaultControls` into the Dimensions panel, so it depends on the values from `DEFAULT_CONTROLS` to be `true`.

Reply:
Ok I've added the suggested changes!